examples/ex10p.cpp

```diff
@@ -45,6 +45,40 @@ using namespace mfem;
 
 class ReducedSystemOperator;
 
+class GeneralResidualMonitor : public IterativeSolverMonitor
```

Comment:
It might be nice to add this simple monitor in regular src since it would be widely applicable to user problems.